FCCee/IDEA/compact/IDEA_o1_v03/FiberDualReadoutCalo_o1_v01.xml

```diff
@@ -594,6 +594,11 @@
       </sipmDim>
       <structure>
         <dim distance="1.5*mm" dx="1.0*mm"/>
+        <!--parameters for neighborhood finding algorithm-->
+        <!--rmax1/rmax2: radius of the neighborhood in units of fiber-fiber distance in barrel/endcap-->
+        <!--x1/x2: margin to incorporate satelite cells at the edge of the tower in barrel/endcap-->
+        <!--delta: flag to remove different type of cells in the neighborhood-->
```

Comment:
can you specify how it is interpreted? Adding e.g. "If set to 0, cherenkov and scintillating fibers will be considered as neighbors". Maybe we could also have a more explicit name than "delta" for this parameter?

Reply:
Added a line of comment as suggested. As for the parameter name, IIUC the name should match one of the available names in the [`DD4hep/DDCore/include/XML/UnicodeValues.h`](https://github.com/AIDASoft/DD4hep/blob/master/DDCore/include/XML/UnicodeValues.h). It won't compile otherwise (correct me if I'm wrong).